app/helpers/user_activity_helper.rb

```diff
@@ -0,0 +1,35 @@
+module UserActivityHelper
+  def activity_selector_week_range
+    start = min_week.max Organization.current.in_preparation_until
```

Comment:
Maybe move this to organization model?
We usually avoid calling Organization.current and do something like:

`def activity_selector_week_range_for(organization = Organization.current)`

and in organization you could do something like:

```
def activity_start_date
   [default_activity_start_date, in_preparation_until].compact.max 
 end
 ```